gtda/diagrams/features.py

```diff
@@ -48,17 +54,30 @@ class PersistenceEntropy(BaseEstimator, TransformerMixin):
     PersistenceImage, PairwiseDistance, Silhouette, \
     gtda.homology.VietorisRipsPersistence
 
+    References
+    ----------
+    .. [1] A. Myers, E. Munch, and F. A. Khasawneh, “Persistent Homology of
+    Complex Networks for Dynamic State Detection,” Apr. 2019,; doi:
```

Comment:
I think this is not correctly indented. Could you take closer inspiration from e.g. VietorisRipsPersistence?

Reply:
I fixed this.